po/ru.po

```diff
@@ -1406,6 +1471,8 @@ msgid ""
 "Android platform development. This includes interoperability with C, C++, and\n"
 "Java."
 msgstr ""
+"[Android Deep Dive](../android.md) - это полудневный курс по использованию Rust \n"
```

Comment:
Nit: Russian typographic tradition suggests using an mdash here. Consider replacing it if the English version is consistently using ndashes in the text (they have a different tradition).

Reply:
I think the English version uses em dash, so I've replaced it with em dash whenever it makes sense